opensrp-giz-malawi/src/main/java/org/smartregister/giz_malawi/util/CrashLyticsTree.java

```diff
@@ -0,0 +1,43 @@
+package org.smartregister.giz_malawi.util;
+
+
+import android.util.Log;
+
+import com.crashlytics.android.Crashlytics;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.smartregister.giz_malawi.application.GizMalawiApplication;
+
+import timber.log.Timber;
+
+public class CrashLyticsTree extends Timber.Tree {
+
+    private static final String CRASHLYTICS_KEY_PRIORITY = "priority";
+    private static final String CRASHLYTICS_KEY_TAG = "tag";
+    private static final String CRASHLYTICS_KEY_MESSAGE = "message";
+
+
+    private String userName;
+
+    @Override
+    protected void log(int priority, @Nullable String tag, @NotNull String message, @Nullable Throwable t) {
+        if (priority == Log.VERBOSE || priority == Log.DEBUG || priority == Log.INFO) {
+            return;
+        }
+
+        if (userName == null) {
+            userName = GizMalawiApplication.getInstance().getContext().allSharedPreferences().fetchRegisteredANM();
```

Comment:
You can move back this tree to core for use by everyone

Reply:
Do you mean the whole file?

Reply:
I guess the only thing that the application needs to do is plant it if they are at that point in their development

Reply:
Moved